tests/test_api_groups_view.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+def test_valid_response(client):
+    response = client.get('/groups/', follow=True)
+    response_keys = response.json().keys()
```

Comment:
I would suggest saving the value of `response.json()` to a variable, and using that in your later code. Also, the `in` keyword used with a dictionary will check whether that dictionary contains that key or not, so you won't need to explicitly call `.keys()` on the JSON.

```python
response_data = response.json()
assert "@id" in response_data
...
```